EnglishFilter/sections/antiadblock.txt

```diff
@@ -587,6 +587,10 @@ uaclips.com,kzclip.com,ru-clip.com,ruclip.com###morev
 !
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/83749
+nyafilmer.*#$#body { overflow: visible!important; }
+nyafilmer.*#?#body > div[class]:has(> h1:contains(Adblock))
+nyafilmer.*#?#body > div[class]:empty:matches-css(position: fixed):matches-css(z-index: /\d+/)
```

Comment:
What are these rules for?

Reply:
I am not sure too. I moved it from the foreign files.

Reply:
I don't see a detection even with VPN. I guess you can remove those rules.